src/libs/type-utils/lodash-fp-types.ts

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line lodash-fp/use-fp -- we need these types from base lodash lib
+import { CurriedFunction2, CurriedFunction3, CurriedFunction4, CurriedFunction5 } from 'lodash'
+
+// variadic types can solve SafeCurry for n arguments, but as of TS v4.8, that technique does not
+// provide the desired type-flow through usages, including curried fns used within _.flow
+
+/** more reliable generic typing for curried functions (with 2 args) */
+export type SafeCurry2<F extends (...args: any[]) => any> =
```

Comment:
Could these use the `AnyFn` type from general-types.ts?

Reply:
yes!  AnyFn was a more recent addition.  I will add it's usage here.

Reply:
done